### ISSUE TITLE
How do I add arrow movement in Firefox file picker?

### ISSUE THREAD

**Issue by AndreasDavour** [k0kubun/xremap#342]
I have just bought a keyboard without arrow keys, and want to modify the keymap so that I can use emacs style C-f and C-b to move backward and forward instead. More precisely, when I right click an image and want to save it, but also edit the file same, I'd like to be able to use C-b to go to the start of the file name and edit it.

I tried this config, but I guess I must be misunderstanding how xremap works, because it does nothing.

```
keymap:
  - name: Emacs binding
    application:
      only: firefox
    remap:
      C-b: left
      C-f: right
      C-p: up
      C-n: down
```

**Comment by k0kubun** [comment]
Does it no work even without the `application:` filter?

**Comment by AndreasDavour** [comment]
You mean just:

```
keymap:
  - name: Emacs binding
    remap:
      C-b: left
      C-f: right
      C-p: up
      C-n: down
```

Just tried that and sadly, no.

**Comment by k0kubun** [comment]
OK. Can you somehow check if the actual left/right/up/down keys work as you expect there? Do you have an alternative keyboard to test it?

If it does work, can you start xremap with `RUST_LOG=debug`, press and release C-p, press and release the actual up, and report the entire output of the xremap here?

**Comment by AndreasDavour** [comment]
I was mystified for a bit why I did not get any output at all when running in debug mode. When I switched to not using sudo, but actually running as root things started happening! Clearly something was up, permissions wise.

I went back and added my user to the correct group and modified the udev rules and then it started to work ok. Now it works like a charm.

Thanks a lot for a very neat tool, and thanks for the patience helping me debug this isssue!
